manifests/b/Beeper/Beeper/3.102.10/Beeper.Beeper.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.4.0 $debug=QUSU.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Beeper.Beeper
+PackageVersion: 3.102.10
+InstallerType: nullsoft # electron-builder
+Scope: user
+InstallerSwitches:
+  Upgrade: --updated
+UpgradeBehavior: install
+Protocols:
+- element
+ReleaseDate: 2024-03-14
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.todesktop.com/2003241lzgn20jd/Beeper%20Setup%203.102.10%20-%20Build%202403272qaonqz6e-x64.exe
+  InstallerSha256: EA97E38DB95565D0093D2C7607A76F61DDDD78B89856BCE7F59FE0059B802CC8
```

Hi @SpecterShell,

The `PackageUrl` led to this `InstallerUrl` - should it be used instead? 

```suggestion
  InstallerUrl: https://download.beeper.com/windows/nsis/x64
  InstallerSha256: CC0671EB158B8AB979A4D28405987FC9B9EB514B1B9EB15F01450EBB3FF82B12
```

---

Probably not? This is a vanity URL and it can quickly become expired as it is updated almost every 2-3 days according to my monitor. 🤔 @stephengillie

---

Out of curiosity, I sent the following to support@beeper.com:

> It looks like the community is attempting to create a Beeper winget manifest (Microsoft's package manager) via https://github.com/microsoft/winget-pkgs/pull/146880 .  However, it looks like there isn't a clear answer on what Beeper's canonical versioned download link should be.  The link on Beeper's website (https://download.beeper.com/windows/nsis/x64) always grabs latest.  Any suggestions on how this can get pointed to a version-specific link?

---

Got a reply saying the above link is the only possible way to get Beeper.  My reply:

> Any suggestions on how to resolve this with Beeper’s help?  The alternative that the community will likely move towards is hosting files externally, which seems less than ideal (having to trust whoever uploads the installers elsewhere).

…and their response:

> We do not provide version specific downloads because of our frequency of breaking changes to the client that correspond with our backend changes.

> Obviously, we want this to get better in the future but as of now we can't help with versioned downloads.

> If the distribution repo accepts non-official download links for apps we have a bigger problem.

…which does seem to indicate either this manifest can’t be for a specific version ( if winget supports that concept?), or it has to point to a non-official download link.